src/lib/database/post.ts

```diff
@@ -407,7 +416,8 @@ export const createPost = async (
         }
       }
     };
-    next(depth);
+    // was this supposed to be awaited?
+    await next(depth);
     return replaceContentWithLinks(post);
```

Comment:
// not to be awaited -- this allows for the initial post to be returned while processing the rest of the post. However, this is being overhauled and this code will change.

Reply:
word